### PR TITLE
Fix typos and formatting

### PR DESCRIPTION
**Pull request by Hedphel**
This is mostly typos and grammar mistakes, although I also took the liberty of changing the format of two pages to make them more consistent overall.

- In `key-bindings` I simply fixed the shortcuts table.

- In `installation-linux.md` I changed the format of Distro-Agnostic AppImage steps to make it more readable.

### CODE REVIEW

**Comment by mmahmoudian** [comment]
Thank you. It seems I had a lot of typos that have slipped my eyes.

I just found two more typos while reading the diff. If you feel like fixing it, go ahead, otherwise I'll fix them in another PR

**Comment by Hedphel** [comment]
I just caught some more typos / grammar mistakes. I think this is now good to go.

**Review by mmahmoudian — APPROVED** [review]

File: content/docs/installation/installation-linux.md [hunk 1]
```diff
@@ -115,16 +116,18 @@ chmod +x Flameshot-*.x86_64.AppImage
 ./Flameshot-*.x86_64.AppImage
 ```
 
-This will create an icon in your system tray area. (usually in the bottom-right ot top-right of the screen). You can now either:
+This will create an icon in your system tray area. (usually in the bottom-right ot top-right of the screen).
```
[the remark this answers is not in the document]
Comment (Hedphel) [reply content/docs/installation/installation-linux.md]: Fixed in the next commit

**Comment by mmahmoudian** [comment]
@Hedphel Thank you

**Pull request merged**